City-of-Helsinki/berth-reservations-admin
# VEN-633 | Populate expandable customer details with values

## Description :sparkles:
Show following data in customer details:
- comment
- berths
- boats
- applications

There will be another ticket that populates data in the table columns.

### Closes :no_good_woman:
https://helsinkisolutionoffice.atlassian.net/browse/VEN-633

### Automated tests :gear:️
- Customer list utils test
- Component render test `CustomerDetails.test.tsx`

### Manual testing :construction_worker_man:
- Go to customers
- Write "toivonen" to search box, you should get one hit
- Expand the row and you should see the values populated

## Screenshots :camera_flash:
![ae267ca1-3c77-4bac-a5f7-4686cfbcdcc8](https://user-images.githubusercontent.com/1407958/83381351-1c8e0b80-a3e9-11ea-8a1b-9bd8751d050c.png)
